hilti/toolchain/src/ast/types/library.cc

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020-2021 by the Zeek Project. See LICENSE for details.
+
+#include "ast/types/library.h"
+
+#include <utility>
+
+#include <hilti/base/util.h>
+
+using namespace hilti;
+using namespace hilti::type;
+
+static std::string normalize(std::string cxx_name) {
+    if ( cxx_name.find(' ') == std::string::npos && ! util::startsWith(cxx_name, "::") )
```

Comment:
What's the `find(' ')` for?

Reply:
This is a dedicated workaround for `__library_type("const hilti::rt::TypeInfo*")`. We can get rid of treatment of spaces if we use `__library_type("hilti::rt::TypeInfo* const")` instead; this seems fine to me as `__library_type` is an internal API anyway.

Reply:
Did just that.